pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -23,13 +23,15 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
-const parallelism = 16
+// Parallelism is used the store value from scheduler config
+var Parallelism = 16
 
 // chunkSizeFor returns a chunk size for the given number of items to use for
 // parallel work. The size aims to produce good CPU utilization.
 func chunkSizeFor(n int) workqueue.Options {
 	s := int(math.Sqrt(float64(n)))
-	if r := n/parallelism + 1; s > r {
+
+	if r := Parallelism + 1; s > r {
```

Comment:
`n/Parallelism`.

It looks like we need a unit test for this function. I don't normally ask for unit tests for non-exported functions, but there is no other way to test the intended chunk size in this case.

Reply:
fun can be compared to `nil` only in go, so how should we plan the UT for this ? The chunk size directly creates the workflow, rather than returning the int. Should I refactor ?

Reply:
refactoring is fine